.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -6,29 +6,27 @@ body:
     id: help
     attributes:
       value: |
-        > **Note**
-        > GitHub Issues use [GitHub Flavored Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for formatting.
+        > Titles should be short, descriptive, and compelling.
   - type: textarea
     id: issue
     attributes:
-      label: Issue Description
+      label: Issue description and context
       description: |
-        Describe the issue you are adding or content you are suggesting.
-        Share any next steps that should be taken our outcomes that would be beneficial.
+        Describe the issue so that someone who wasn't present for its discovery can understand the problem and why it matters. Use full sentences, plain language, and good [formatting](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax – links and images welcome! Share desired outcomes or potential next steps.
```

Comment:
> Describe the issue so that someone who wasn't present for its discovery can understand the problem and why it matters. Use full sentences, plain language


I really like this new language

> links and images welcome! Share desired outcomes or potential next steps.

I would however prefer we keep the links and images in their own section. That would keep the structure of the issues consistent.

Reply:
Note that it doesn't say "link to related _issues_", and I modified the language below this from `Issue Links` to `Other issues`(with amended description text). I do want to encourage people to link to relevant content that isn't an issue in this section.

>  That would keep the structure of the issues consistent.

Issues are templates, but templates shouldn't be dogma or otherwise inflexible. Our team can elect to keep the structure, but I want to include "ease of use for not-our-team" in how we're optimizing.

Reply:
Tried to make this clearer in another revision